src/styles/blocks/icons.scss

```
@@ -0,0 +1,78 @@
+.logo {
+  &__img {
+    height: 46px;
+
+    @include onTablet {
+      height: 64px;
+    }
+  }
+}
+
+.icon {
+  display: block;
+  height: 18px;
+  width: 18px;
+
+  @include onTablet {
+    height: 24px;
+    width: 24px;
+  }
+
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  &__details {
+    display: none;
+  }
+
+  &--menu {
+    background-image: url("../images/menu.svg");
+    margin-left: 30px;
+
+    @include onTablet {
+      margin-left: 63px;
+    }
+
+    &:hover {
+      background-image: url("../images/menu_hover.svg");
+    }
+  }
+
+  &--call {
+    background-image: url("../images/phone.svg");
+    text-decoration: none;
+  }
+
+  &--cross {
+    background-image: url("../images/cross.svg");
+  }
+
+  @include onDesktop {
+    &__details {
+      display: block;
+      width: max-content;
+      position: relative;
+      transform: translateX(-100%);
+      padding-right: 20px;
+      top: -40px;
```

```suggestion
      transform: translate(-100%, -50%);
      padding-right: 20px;
```

do not use relative on the element you want to position